README.md

```diff
@@ -37,9 +39,11 @@
 - [Commit Messages](/standards/commits.md) at Bitwise
 
 #### Code Versioning
+
 - [Bitwise SemVer](/standards/code-versioning.md)
 
 #### Project Setup
+
```

Comment:
Thank you for making our markdown consistent (ie linting)! Looks much better looking at the source.